### PR TITLE
docs: add story for < TokenTable />

### LINKED ISSUE #90: docs: Create basic story for < TokenTable />
[Monday ticket](https://monday.monday.com/boards/3532714909/pulses/5350319978)

Add basic story by structure similar to existing ones

- https://github.com/mondaycom/vibe-storybook-components/blob/master/CONTRIBUTING.md
- https://github.com/mondaycom/vibe-storybook-components/blob/master/COMPONENTS_DOCUMENTATION_GUIDELINES.md

### PR DESCRIPTION
**Pull request by Dhoni77**
Closes #90

Added story for `<TokenTable/>` component.

### CODE REVIEW

**Comment by Dhoni77** [comment]
cc @SergeyRoyt

File: src/components/token-table/__stories__/token-table.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Controls, Canvas, Meta } from '@storybook/blocks';
+import * as TokenTable from './token-table.stories';
+
+<Meta of={TokenTable} />
+
+# TokenTable
+
+## Overview
+
+TokenTable component is used to display a table with rows and columns.
```
Comment (SergeyRoyt) [inline src/components/token-table/__stories__/token-table.mdx:10]:
```suggestion
TokenTable component is used to display a table with rows and columns and is designed to display tokens values. 
```

File: src/components/token-table/__stories__/token-table.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  component: TokenTable,
+  title: 'Components/TokenTable',
+};
+export default meta;
+
+type Story = StoryObj<typeof TokenTable>;
+
+export const Overview: Story = {
+  args: {
+    theadData: ['Column A', 'Column B'],
```
Comment (SergeyRoyt) [inline src/components/token-table/__stories__/token-table.stories.tsx:14]:
```suggestion
    theadData: ['Token', 'Value'],
```

We are just mostly using it to display smth like this
image

File: src/components/token-table/__stories__/token-table.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    tbodyData: [
+      { id: '1', items: ['value 1', 'value 3'] },
+      { id: '2', items: ['value 2', 'value 4'] },
```
Comment (SergeyRoyt) [inline src/components/token-table/__stories__/token-table.stories.tsx:17]:
```suggestion
      { id: '1', items: ['--token-1', '#111'] },
      { id: '2', items: ['--token-2', '#222'] },
```

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
Nice!
Let's just adjust example a little bit to be more specific to out use case

**Review by SergeyRoyt — APPROVED** [review]
Amazing! Thanks a lot!

**Pull request merged**